client/app/bundles/SearchPage/components/searchBox.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component, PropTypes } from 'react';
+import updateSearchValue from '../actions/updateSearchValue';
+import performSearch from '../actions/performSearch';
+
+export default class SearchBox extends Component {
+  static propTypes = {
+    value: PropTypes.string.isRequired,
+    dispatch: PropTypes.func.isRequired
+  }
+
+  constructor() {
+    super()
+
+    _.bindAll(this, 'onInputChange', 'performSearch')
```

Comment:
I didn't bother commenting this because it has nothing new in it. Comments in other file cover anything in this file